deploy/docker/Dockerfile

```diff
@@ -1,28 +1,44 @@
 FROM alpine:3.5
 
-LABEL maintainer="adundovi@gmail.com"
-LABEL description="Prepares a docker image \
-of pump.io server, http://pump.io/."
+LABEL maintainer Andrej Dundovic <adundovi@gmail.com>
+LABEL description "Stream server that does \
+most of what people really want from a social network."
 
 ENV LC_ALL en_US.UTF-8
 
+ENV PUMPIO_VERSION 3.0.2
+
 RUN apk add --no-cache \
-  nodejs \
-  git \
-  python \
-  make \
-  g++ \
-  graphicsmagick
-RUN npm install -g databank-mongodb
-RUN npm install -g pump.io
-RUN adduser -D -g "" pumpio
+    curl \
+    nodejs \
+    python \
+    make \
+    g++ \
+    git \
+    graphicsmagick \
+ && adduser -D -g "" pumpio \
+ && mkdir -p /etc/ssl/private \
+ && npm install -g databank-mongodb@0.19.2 \
+ && mkdir -p /usr/src \
+ && curl -fSL https://github.com/pump-io/pump.io/archive/v$PUMPIO_VERSION.zip -o pumpio.zip \
```

Comment:
Again, I'd prefer for this to be built from the currently checked-out tree. That lets people use Docker for development, plus downloading archives from GitHub when you already have a local copy feels kind of sketchy.

(That and the fact that this will probably break my release workflow - ideally we want to automatically publish Docker images whenever a new release comes out, which means automatically doing it during `npm version` or `npm publish` or something, when I might not have pushed the tag yet.)